### PR TITLE
Feature/input

### PR DESCRIPTION
**Pull request by WettyWednesday**

### CODE REVIEW

**Review by ThePhoneBeLinging — CHANGES_REQUESTED** [review]

File: src/main/java/Group15/Api/Api.java [hunk 1]
```diff
@@ -10,28 +10,24 @@
...
 public class Api
 {
-    public static void getWorkout()
-    {
-        new Thread(() ->
-            {
-            String URL = "https://158.179.205.63/api/v3/get/exercise/byBodyPart?bodyPart=Quadriceps&amountOfExercises=5";
-            try
-            {
+    public static void getExercisesFromBodypart(String bodyPart, int exerciseAmout) {
```
Comment (ThePhoneBeLinging) [inline src/main/java/Group15/Api/Api.java:13]: Still doesn't return the exercises

File: src/main/java/Group15/ViewController.java [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 public class ViewController
 {
     private static Stage stage;
 
-    public static void init()
+    public static void init(Stage stage)
     {
-        ViewController.stage = new Stage();
-        ViewController.stage.show();
+        stage.setScene(createMuscleSelectorScene());
```
Comment (ThePhoneBeLinging) [inline src/main/java/Group15/ViewController.java:21]: What happens the next time someone tries to use the setScene or getScene method? :)

**Review by ThePhoneBeLinging — CHANGES_REQUESTED** [review]

File: src/main/java/Group15/Api/Exercise.java [hunk 3]
```diff
@@ -12,4 +12,10 @@ public class Exercise
     public Exercise()
     {
     }
+
+    @Override
+    public String toString() {
+        return "Title: " + title + "\n" + "Description: " + description + "\n" + "Type: " + type + "\n" + "Body Part: "
+                + bodyPart + "\n" + "Equipment: " + equipment + "\n" + "Difficulty: " + difficulty;
+    }
```
Comment (ThePhoneBeLinging) [inline src/main/java/Group15/Api/Exercise.java]: Useless feature

**Review by ThePhoneBeLinging — APPROVED** [review]

File: src/main/java/Group15/ViewController.java [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
 {
     private static Stage stage;
 
-    public static void init()
+    public static void init(Stage stage)
     {
-        ViewController.stage = new Stage();
-        ViewController.stage.show();
+        ViewController.stage = stage;
+        stage.setScene(createMuscleSelectorScene());
```
Comment (emilsavino) [inline src/main/java/Group15/ViewController.java:22]: The scene should be set to the homeScreen

**Review by emilsavino — APPROVED** [review]

**Pull request merged**